.github/pull_request_template.md

```diff
@@ -28,6 +29,8 @@ Please describe the tests you've added or any tests that already cover this chan
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] I have added any new baselines to the repo
+- [ ] I have assigned reviewers (see [VisIt's PR procedures](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/pr_create.html#reviewers) for more information).
 
 
-*Don't forget to squash merge when this pull request is approved*
+*Don't forget to squash merge when this pull request is approved.*
```

Comment:
I wonder if we (members and owners) should be the only ones doing the merge to develop. If so, lets remove this line.

Reply:
@visit-dav/visit-developers I think this is the right way to notify everyone: what does everything think about owners and members being the ones who squash-merge the PR's?

Reply:
So, I read this content as though it is intended to be instruction to the _submitter_ of the PR and not the instructions to the owners/members of the repo on how to handle PRs. If the former is correct, then I think this instruction is confusing because the submitter will not be able to do the merge anyways (squash merge or not) unless s/he is also an owner/member. 

I regard the act of merging PR as separate from submission and review even if all three wind up being performed by the same owner/member in regular instances of the workflow.

Reply:
I think it makes more sense to remove this comment, so I did.